test/util/cart_utils.py

```diff
@@ -278,10 +281,18 @@ def launch_test(self, cartobj, cmd, srv1=None, srv2=None):
     def launch_cmd_bg(self, cartobj, cmd):
         """ launches the given cmd in background """
 
+        self.print_cmd("\nCMD : %s\n" % cmd)
+
         cmd = shlex.split(cmd)
         rtn = subprocess.Popen(cmd)
 
         if rtn is None:
             return -1
 
         return rtn
+
+    def print_cmd(self, cmd):
+        stdout = logging.getLogger('avocado.test.stdout')
```

Comment:
while not critical, probably best to get stdout/logfile once during init of this class, and refer to self.stdout/self.logfile instead of querying each time

Reply:
Done